types/function_list_builder.go

```diff
@@ -51,19 +53,47 @@ func (s *FunctionLookupBuilder) Build() (map[string][]string, error) {
 		return serviceMap, marshalErr
 	}
 
+	topicDelim := ","
+
+	if delim, exists := os.LookupEnv("topic_delimiter"); exists && len(delim) > 0 {
+		topicDelim = delim
```

Comment:
I would suggest to trim spaces here. Otherwise I am wondering would we like to make this configurable? Is there a case where a user would use `,` as part of the topic name?

Reply:
Moved the lookup out into the consuming connector